ezbake/config/conf.d/auth.conf

```diff
@@ -1,39 +1,116 @@
 authorization: {
-    # TODO: Fill these in with real default rule definitions - SERVER-846.
-    # Putting just the ones for the certificate authority in for now since
-    # these are always evaluated by trapperkeeper-authorization and are
-    # needed to complete a successful agent run.
+    version: 1
     rules: [
         {
+            # Allow nodes to retrieve their own catalog
+            match-request: {
+                path: "^/puppet/v3/catalog/([^/]+)$"
+                type: regex
+                method: [get, post]
+            }
+            allow: "$1"
+            sort-order: 500
+            name: "puppetlabs catalog"
+        },
+        {
+            # Allow nodes to retrieve the certificate they requested earlier
+            match-request: {
+                path: "/puppet-ca/v1/certificate/"
```

Comment:
This rule covers both the CA and everything else.  Since the rule was exactly the same, we don't actually need to represent it twice like the old one does: https://github.com/puppetlabs/puppet/blob/master/conf/auth.conf#L102-L113